tests/python/dlight/test_gpu_matmul.py

```diff
@@ -28,7 +28,7 @@ class BaseBeforeAfter(tvm.testing.CompareBeforeAfter):
     @pytest.fixture
     def transform(self):
         def transform(mod):
-            with Target("nvidia/geforce-rtx-3090-ti"):
+            with Target("nvidia/geforce-gtx-1080-ti"):
```

Comment:
The target tag here should not influence codegen so I suggest keeping it to its original value.

Reply:
I changed this tag since the target "nvidia/geforce-gtx-1080-ti" has low SM version and can not apply tensorization. With this tag, tensorization rule will not be applied, which matches the original functionality of this test file.